src/Helpers/Bitcoin.test.ts

```diff
@@ -0,0 +1,81 @@
+import { describe } from 'riteway'
+import * as bitcore from 'bitcore-lib'
+import { getPoetTimestamp } from './Bitcoin'
+
+const createTransaction = (prefix: string) => {
+    // All data is a mock
+    const privateKey = new bitcore.PrivateKey('L1uyy5qTuGrVXrmrsvHWHgVzW9kKdrp27wBC7Vs6nZDTF2BRUVwy');
+    const utxo = [new bitcore.Transaction.UnspentOutput({
+      "txId" : "115e8f72f39fad874cfab0deed11a80f24f967a84079fb56ddf53ea02e308986",
+      "outputIndex" : 0,
+      "address" : "17XBj6iFEsf8kzDMGQk5ghZipxX49VXuaV",
+      "script" : "76a91447862fe165e6121af80d5dde1ecb478ed170565b88ac",
+      "satoshis" : 50000
+    })];
+  
+    return new bitcore.Transaction()
+      .from(utxo)
+      .to('1Gokm82v6DmtwKEB8AiVhm82hyFSsEvBDK', 15000)
+      .addData(new Buffer(`${prefix}0000ipfsDirectoryHash`))
+      .sign(privateKey);
+}
+
+describe('Bitcoin', async (should: any) => {
+  const { assert } = should('')
```

Comment:
@kennylavender why are we always passing empty string to should? I've seen this in every PR but I'm not sure whether this is the way to do it

Reply:
@lautarodragan The should call provides a default *should* value for all the the assertions. Its not necessary to provide a string if there is not a common *should* between the assertions. I have a habit of typing an empty string. There are no issues with passing empty strings as far as I know, but if we notice empty strings being passed to should it would probabaly be good to remove them for cleanliness and to avoid any confusion.